Keda.Cosmosdb.Scaler/src/Services/ExternalScalerService.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Threading.Tasks;
+using Google.Protobuf.Collections;
+using Grpc.Core;
+using Keda.Cosmosdb.Scaler.Protos;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.ChangeFeedProcessor;
+using Microsoft.Azure.Documents.Client;
+
+namespace Keda.Cosmosdb.Scaler.Services
+{
+    public class ExternalScalerService : ExternalScaler.ExternalScalerBase

```

Comment:
Let's use some better names here
```suggestion
    public class CosmosDbExternalScaler : ExternalScaler

```